test/allCreateRecordsTest.js

```diff
@@ -71,12 +72,92 @@ describe('my Testing suite', () => {
         done(error);
       });
   });
+  
+  it('users should be able to create record with images and videos when provide token', (done) => {
+    router()
+      .post('/api/v1/red-flags/')
+      .set('Authorization', adminToken)
+      .set('Content-Type', 'application/x-www-form-urlencoded')
+      .field('title', 'Bad road')
+      .field('type', 'intervation')
+      .field('comment', 'testo')
+      .attach('images', 'server/v1/api/models/uploadedFiles/imga.png')
+      .attach('videos', 'server/v1/api/models/uploadedFiles/sample.mp4')
+      .end((error, response) => {
+        expect(response).to.have.status([201]);
+        expect(response.body).to.be.a('object');
+        expect(response.body).to.have.property('status');
+        expect(response.body).to.have.property('message');
+        expect(response.body).to.have.property('record');
```

Comment:
Hello, @key-joshua please make sure that you also test the value of the field/key. Testing the presence of the key is good, but ensuring the value of the key meets expectations is better.